drivers/sensor/vl53l1x/vl53l1_platform_user_data.h

```diff
@@ -0,0 +1,83 @@
+/* vl53l1_platform_user_data.h - Zephyr customization of ST vl53l1x library. */
+
+/*
+ * Copyright (c) 2017 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_VL53L1X_VL53L1_PLATFORM_USER_DATA_H_
+#define ZEPHYR_DRIVERS_SENSOR_VL53L1X_VL53L1_PLATFORM_USER_DATA_H_
+
+#include "vl53l1_def.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @struct VL53L1_Dev_t
+ * @brief  Generic PAL device type that does link between API and platform
+ * abstraction layer
+ *
+ */
+typedef struct {
```

Comment:
Ah, please don't introduce new typedefs. A simple struct pointer should work.

Reply:
See my other comment, but I should be able to remove this typedef. It's only the `VL53L1_DEV` that is required by the STM library, I believe.